### ISSUE TITLE
index.js:6: SyntaxError: Unexpected token [

### ISSUE THREAD

**Issue by konstunn** [arshadkazmi42/squash#54]
After installing with npm and running squash I have the following error
```
konstunn@asus:~ (master)$ squash
/usr/local/lib/node_modules/ak-squash/index.js:6
const [, , ...args] = process.argv;
      ^

SyntaxError: Unexpected token [
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:374:25)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Function.Module.runMain (module.js:442:10)
    at startup (node.js:136:18)
    at node.js:966:3
```

**Comment by arshadkazmi42** [comment]
can you share you `node` and `npm` versions

**Comment by konstunn** [comment]
Yes, sure thing, that would make more sense to it.

```
konstunn@asus:~ (master)$ node --version
v4.2.6
konstunn@asus:~ (master)$ npm --version
3.5.2

```

**Comment by arshadkazmi42** [comment]
This library will work for node 8+

**Comment by konstunn** [comment]
Woah, it seems that I have pretty ancient node version. :-)

Or not. It seems that major version 4 dates back to March 2018. So I propose to track this node version 8+ dependency in your project.

**Comment by arshadkazmi42** [comment]
> So I propose to track this node version 8+ dependency in your project somehow.

After seeing this, I was thinking for including dependencies version somewhere in the readme.

**Comment by arshadkazmi42** [comment]
@konstunn Can you upgrade to latest version of node and try this. So to confirm that its node version issue and then we can close this issue

**Comment by arshadkazmi42** [comment]
Fixed in #55
